test/testdata/reference/t_multiline_file.js

```diff
@@ -0,0 +1,12 @@
+setup({
+  single_test: true,
+  explicit_done: false
+});
+const value = 'value';
+assert_equals(value, 'value');
+assert_not_equals(value.length, badValue);
```

Comment:
Does it matter that 'badValue' is undefined?

Reply:
Ultimately, no, since the transformation doesn't care. If anything I don't necessarily _want_ this file to compile nicely since it's just meant to be play code for the test. I can add a definition for it if you think that would be more clear.

Reply:
Yeah, either define it or add a comment as to why it's not defined - just to avoid confusing the reader.